### PR TITLE
add new speaker in BR

### PR DESCRIPTION
**Pull request by ErickWendel**
Signed-off-by: Erick Wendel <erick.workspace@gmail.com>

### CODE REVIEW

**Review by ZYSzys — APPROVED** [review]

File: locale/en/get-involved/node-speakers.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 
 ## Speakers
 
+### Brazil 🇧🇷
+
+#### São Paulo
+
+##### São Paulo
```
Comment (ghost) [inline locale/en/get-involved/node-speakers.md:24]: Are you sure this duplication is OK with 'São Paulo'?

Comment (ZYSzys) [reply locale/en/get-involved/node-speakers.md:24]: IIUC, the first means `state`, and second means `city`.

Refs: https://github.com/nodejs/nodejs.org/pull/2005#pullrequestreview-196402720

Comment (ghost) [reply locale/en/get-involved/node-speakers.md:24]: @ZYSzys：So 'São Paulo' is a state and a city?

Comment (ZYSzys) [reply locale/en/get-involved/node-speakers.md:24]: Maybe yes.
screen shot 2019-02-10 at 12 24 43 pm

Refs: https://en.wikipedia.org/wiki/S%C3%A3o_Paulo

Comment (ghost) [reply locale/en/get-involved/node-speakers.md:24]: OK

**Review by ghost — APPROVED** [review]

**Pull request merged**